.github/workflows/main.yml

```
@@ -32,34 +27,25 @@ jobs:
-              - 'tests/cypress/**'
-            playwright:
-              - 'tests/playwright/**'
-            robot:
-              - 'tests/robot/**'
-            selenium:
-              - 'tests/selenium/**'
-            k6:
-              - 'tests/k6/**'
 
   backend-tests:
+    if: needs.filter.outputs.backend == 'true'
```

actionlint (1.7.7)

property "filter" is not defined in object type {}

(expression)